### PR TITLE
Feature/trainings save

### PR DESCRIPTION
**Pull request by markirius**

### CODE REVIEW

File: bothub/api/v2/repository/views.py [hunk 1]
```diff
@@ -843,6 +843,61 @@ def word_suggestions(self, request, **kwargs):
 
         return Response({"suggestions": suggestions})
 
+    @action(
```
Comment (dyohan9) [inline bothub/api/v2/repository/views.py:846]: explain to me what this route does

File: bothub/api/v2/repository/views.py [hunk 2]
```diff
@@ -843,6 +843,61 @@ def word_suggestions(self, request, **kwargs):
...
+        try:
+            repository = get_object_or_404(
+                Repository, pk=self.request.data.get("repository")
+            )
+        except DjangoValidationError:  # pragma: no cover
+            raise PermissionDenied()
+
+        try:
+            repository_version = get_object_or_404(
+                RepositoryVersion, pk=self.request.data.get("repository_version")
```
Comment (dyohan9) [inline bothub/api/v2/repository/views.py:862]: why do the same consultation twice? you could simplify this query using filters

File: bothub/api/v2/repository/views.py (hunk 1 above)
Comment (markirius) [reply bothub/api/v2/repository/views.py:846]: this route save more than one training in same time, passing repository, repository_version and examples to be added in a list value with examples to be add, like:
`{ "repository": uuid,
  "repository_version": pk, 
  "examples": [{
                        "text": "yes",
                        "language": "en",
                        "entities": [{
                            "label": "yes",
                            "entity": "_yes",
                            "start": 0,
                            "end": 3
                        }],
                        "intent": "greet"
                    }, 
                   {
                        "text": "yes",
                        "language": "en",
                        "entities": [{
                            "label": "yes",
                            "entity": "_yes",
                            "start": 0,
                            "end": 3
                        }],
                        "intent": "greet"
                    },
       ]
 }`

Comment (markirius) [reply bothub/api/v2/repository/views.py:846]: this content body will be processed by this route, saving the list of examples

File: bothub/api/v2/repository/views.py (hunk 2 above)
Comment (markirius) [reply bothub/api/v2/repository/views.py:862]: in that case it's needed an repository_version object to be added on get_or_create a intent

**Pull request closed without merging**